src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBOptions.java

```diff
@@ -35,12 +36,16 @@ public GenomicsDBOptions(final Path reference, final GenomicsDBArgumentCollectio
         this.useBCFCodec = genomicsdbArgs.useBCFCodec;
         this.sharedPosixFSOptimizations = genomicsdbArgs.sharedPosixFSOptimizations;
         this.useGcsHdfsConnector = genomicsdbArgs.useGcsHdfsConnector;
+        if (genomicsdbArgs.maxDiploidAltAllelesThatCanBeGenotyped - 1 < genotypeCalcArgs.maxAlternateAlleles) {
+            throw new UserException.BadInput("GenomicsDB max alternate alleles (" + GenomicsDBArgumentCollection.MAX_ALTS_LONG_NAME
+                + ") must be at least one greater than genotype calculation max alternate alleles ("
+                + GenotypeCalculationArgumentCollection.MAX_ALTERNATE_ALLELES_LONG_NAME + "), accounting for the non-ref allele");
+        }
+        this.maxDiploidAltAllelesThatCanBeGenotyped = genomicsdbArgs.maxDiploidAltAllelesThatCanBeGenotyped;
         if (genotypeCalcArgs != null) {
-            this.maxDiploidAltAllelesThatCanBeGenotyped = genotypeCalcArgs.maxAlternateAlleles;
             this.maxGenotypeCount = genotypeCalcArgs.maxGenotypeCount;
```

Comment:
Did we determine whether we also need a separate max genotype count arg for GenomicsDB? Does the GenomicsDB value also need to be greater than the value for genotyping? @mlathara opinions?

Reply:
Using IntelliJ it looks like it's just HaplotypeCaller.